src/main/java/seedu/mtracker/commons/Validate.java

```diff
@@ -101,6 +102,15 @@ public static void validateIndexWithinBounds(ArrayList<Instrument> instruments,
         }
     }
 
+    public static void checkIsNotDone(ArrayList<Instrument> instruments, int instrumentNumber)
+            throws AlreadyDoneError {
+        Instrument instrument = instruments.get(instrumentNumber);
+        boolean currentStatus = instrument.getIsDone();
```

Comment:
```suggestion
        boolean isDoneStatus = instrument.getIsDone();
```

to make it sound more boolean